did.go

```diff
@@ -126,6 +195,52 @@ var DIDMethodNetwork = map[DIDMethod]map[DIDNetworkFlag]byte{
 	},
 }
 
+// DIDMethodNetworkOpts is a structure to represent DID method network options
+type DIDMethodNetworkOpts struct {
```

Comment:
DIDMethodNetworkOpts => DIDMethodNetworkParams
move optional params to func() RegistrationOption